src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByRefreshTokenParameterBuilder.cs

```diff
@@ -71,7 +71,8 @@ internal override ApiEvent.ApiIds CalculateApiEventId()
         public AcquireTokenByRefreshTokenParameterBuilder WithSendX5C(bool withSendX5C)
         {
             CommonParameters.AddApiTelemetryFeature(ApiTelemetryFeature.WithSendX5C);
-            Parameters.SendX5C = withSendX5C;
+            Parameters.SendX5C = withSendX5C; 
+            Parameters.SetPerRequestX5C = true;
```

Comment:
Can we try to avoid having 2 params that do the same thing. There is an AppConfig and there are request params.